### DISCUSSION TITLE
Custom Android instructions outdated?

### DISCUSSION THREAD

**Discussion by caseygibson-emesent** [react-native-webview/react-native-webview#3611] [Q&A]
Hi, 

I tried following the instructions found at https://github.com/react-native-webview/react-native-webview/blob/master/docs/Custom-Android.md but they throw Java errors as the overridden method `createViewInstance` is different. It's now returning a `RNCWebViewWrapper`.

![image](https://github.com/user-attachments/assets/847f44f4-9f1d-4a73-84c8-03e09063cf8b)

Am I missing something?

**Comment by caseygibson-emesent** [comment]
The link in the docs for "register the module" also doesn't work, so it would have been better if this was part of the example project so people can see exactly how this works even if the docs are abandoned.

**Comment by keima** [comment]
Due to the way New Architecture works, it seems that at least the UI props and commands (methods of the View that are exposed via `ref`) and the methods of NativeModule can no longer be implemented differently (extends) in RNCWebViewModule/Manager.

This does not seem to be a problem that can be solved with react-native-webview alone.
https://github.com/facebook/react-native/blob/4165884b700341ba549ed8385480fd98128e7bb1/packages/react-native-codegen/src/parsers/parsers-commons.js#L1077

**Reply by keima** [reply]
I am currently trying to implement a custom webview using react-native-webview v13.12. If I find anything out, I would like to share it.
Also, if anyone knows anything, please let me know.
